tests/fulldecay/example_decay_chains.py

```diff
@@ -0,0 +1,96 @@
+# A D+ particle with only one way of decaying
+dplus_single = {
+    "D+": [

```

Comment:
These dicts can be a bit dry to read, as it were. It would make sense to me to have a docstring or a comment saying how they can easily be obtained with DecayLanguage. So for example:
```
from decaylanguage import DecayMode, DecayChain
dm1 = DecayMode(1, 'K- pi+ pi+ pi0', model='PHSP', zfit="rel-BW")
dm2 = DecayMode(1, 'gamma gamma')
dc = DecayChain('D+', {'D+':dm1, 'pi0':dm2})

This is easier to read and understand at a quick glance :-).
dc.to_dict()
```

Reply:
Furthermore, we guarantee by this the compatibility to DecayLanguage: if the format of it changes, this test will (as it should) break

Reply:
I can replace these dicts with a `DecayChain` if particles only decay in one way but I thought there was no way than using dicts in DecayLanguage to represent a decay with multiple ways of decaying, such as [`pi0_4branches`](https://github.com/zfit/phasespace/pull/63/files#diff-b1d887000ad7868d242b462d007f6e2df795c5ebde3b36ec86b0f81e29f1e134R19-R26). In those cases, should I replace subsets of the decay with `DecayChain`s?

Reply:
No. what is not possible is to define multiple decay chains via the classes. That can be seen as a limitation. The dict representation can do "anything" - remember that you can parse a decay file of any complexity and "translate" the full set of decay chains into a dict. For your pi0 case there's actually an example in the README, which is
```
s = """Decay pi0
0.988228297   gamma   gamma                   PHSP;
0.011738247   e+      e-      gamma           PI0_DALITZ;
0.000033392   e+      e+      e-      e-      PHSP;
0.000000065   e+      e-                      PHSP;
Enddecay
"""

dfp = DecFileParser.from_string(s)
dfp.parse()
```
You can maybe give this one as an example and leave 1-2 of the other dicts. My main point is that we should simplify where possible. Makes sense?

Reply:
That makes sense. Thanks!

Reply:
Fixed in 96aea999fa3bc10a04b8e9e2a24030d294d2ac24